internal/pkg/template/templates/addons/aurora/env/rdws/ingress.yml

```diff
@@ -0,0 +1,28 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
```

Comment:
Q: should this wording be different for RDWS?

Reply:
done!